src/app/modules/features/components/payment/payment.module.ts

```diff
@@ -0,0 +1,34 @@
+// import { paymentRoutes } from './payment.routing';
+// import { PinPaymentComponent } from './pin-payment/pin-payment.component';
+// import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+// import { NgModule } from '@angular/core';
+// import { CommonModule } from '@angular/common';
+// import { RouterModule } from '@angular/router';
+// import { HttpClientModule } from '@angular/common/http';
+// import { NgxSpinnerModule } from 'ngx-spinner';
+// import { AngularFontAwesomeModule } from 'angular-font-awesome';
+// import { TokenizedCardComponent } from 'src/app/modules/features/components/payment/tokenized-card/tokenized-card.component';
+// import { PinValidateComponent } from 'src/app/modules/features/components/payment/pin-validate/pin-validate.component';
+// import { InternationalPaymentComponent } from 'src/app/modules/features/components/payment/international-payment/international-payment.component';
+// import { InputErrorsComponent } from 'src/app/modules/shared/input-errors/input-errors.component';
+//
+//
+// @NgModule({
+//   imports: [
+//     CommonModule,
+//     FormsModule,
+//     ReactiveFormsModule,
+//     RouterModule.forChild(paymentRoutes),
+//     HttpClientModule,
+//     NgxSpinnerModule,
+//     AngularFontAwesomeModule,
+//   ],
+//   declarations:
+//     [TokenizedCardComponent,
+//       PinValidateComponent,
+//       PinPaymentComponent,
+//       InternationalPaymentComponent,
+//       InputErrorsComponent],
+//
+// })
+// export class PaymentModule { }
```

Comment:
What happened to this file? Is it no longer used? I think you can remove it instead of commenting on everything

Reply:
we nolonger use it... i had made sure that we have fewer modules that will hold our components according to the structure i made... thanks for pointing this out... had not seen it